frontends/concrete-python/examples/tfhers-cml/test.py

```diff
@@ -0,0 +1,168 @@
+import os
+from functools import partial
+
+import click
+import numpy as np
+
+from concrete import fhe
+from concrete.fhe import tfhers
+
+from numpy.random import randint
+
+# Params: users shouldn't change them
+LWE_DIM = 909
+GLWE_DIM = 1
+POLY_SIZE = 4096
+PBS_BASE_LOG = 15
+PBS_LEVEL = 2
+MSG_WIDTH = 2
+CARRY_WIDTH = 3
+ENCRYPTION_KEY_CHOICE = tfhers.EncryptionKeyChoice.BIG
+LWE_NOISE_DISTR = 0
+GLWE_NOISE_DISTR = 2.168404344971009e-19
+
+assert GLWE_DIM == 1, "glwe dim must be 1"
```

Comment:
should we hide this and make it constant to users, if they shouldn't change it?

Reply:
But our users will use constant TFHE-rs parameters, or not? Is it expected that TFHE-rs users change parameters?

Reply:
Yes, it's is expected.

Reply:
Hum, I'd like to discuss that part, I keep it for the weekly. Because, I mean, we have a lot of users who are far from being powerusers, they just want the simplest things.